/**:
  ros__parameters:
    K_x: 3.0
    DETECT_CLASS1 : "RedStar"
    DETECT_CLASS2 : "BlueCylinder"
